src/config.type.ts

```diff
@@ -5,6 +5,11 @@ import {InformationMessage} from './modules/information/information.js'
 export type Config = {
 	guildId: string,
 	clientId: string,
+	poll: {
+		emojiId: string,
+		yesEmojiId: string,
+		noEmojiId: string
+	} | null,
```

Comment:
why `null`? if this is optional then it should be `poll?: { ... }` which is shorthand for `{ ... } | undefined`